pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/kubelet/cliflags.go

```diff
@@ -41,6 +42,17 @@ func CLIFlags(kubernetesVersion *semver.Version, criName extensionsv1alpha1.CRIN
 		fmt.Sprintf("--node-labels=%s=%s", v1beta1constants.LabelWorkerKubernetesVersion, kubernetesVersion.String()),
 	)
 
+	// maps are unsorted in go, make sure to output node labels in the exact same order every time
+	labelKeys := make([]string, 0, len(nodeLabels))
+	for key := range nodeLabels {
+		labelKeys = append(labelKeys, key)
+	}
+	sort.Strings(labelKeys)
```

Comment:
Why do we need sorted order of labels?

Reply:
Otherwise, the OSC will likely change on every reconciliation, although semantically nothing changed.
I think we should avoid this

Reply:
In that case, can you extend with a comment on why sorting is required, cause just looking into this part of the code, it seems redundant?